test/fixtures/lightscript/named-arrow-functions/skinny-white-empty/expected.json

```diff
@@ -0,0 +1,123 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 9,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 2,
+      "column": 1
+    }
+  },
+  "program": {
+    "type": "Program",
+    "start": 0,
+    "end": 9,
+    "loc": {
+      "start": {
+        "line": 1,
+        "column": 0
+      },
+      "end": {
+        "line": 2,
+        "column": 1
+      }
+    },
+    "sourceType": "script",
+    "body": [
+      {
+        "type": "NamedArrowDeclaration",
+        "start": 0,
+        "end": 7,
+        "loc": {
+          "start": {
+            "line": 1,
+            "column": 0
+          },
+          "end": {
+            "line": 1,
+            "column": 7
+          }
+        },
+        "id": {
+          "type": "Identifier",
+          "start": 0,
+          "end": 2,
+          "loc": {
+            "start": {
+              "line": 1,
+              "column": 0
+            },
+            "end": {
+              "line": 1,
+              "column": 2
+            },
+            "identifierName": "fn"
+          },
+          "name": "fn"
+        },
+        "generator": false,
+        "expression": false,
+        "async": false,
+        "params": [],
+        "skinny": true,
+        "body": {
+          "type": "BlockStatement",
+          "start": 5,
+          "end": 7,
```

Comment:
Yes:

https://github.com/wcjohnson/babylon-lightscript/blob/a27269694673bbe26aed026fcdbc23ecc64ca94f/test/fixtures/lightscript/named-arrow-functions/skinny-white/expected.json

`parseWhiteBlock` is calling `startNode` before `eat(tt.Arrow)`, so the arrow's position is mapped into the resulting whiteblock node.